### ISSUE TITLE
Formatting breaks my variable names

### ISSUE THREAD

**Issue by AlephAlpha** [microsoft/vscode-cpptools#5929]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Manjaro Linux 20.0.3
- VS Code Version: Code - OSS 1.47.1
- C/C++ Extension Version: v0.30.0-insiders
- Other extensions you installed (and if the issue persists after disabling them):
- Does this issue involve using SSH remote to run the extension on a remote machine?: No
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

**Steps to reproduce**

1. In extension settings, make sure that `clang_format_style` or `clang_format_fallbackStyle` is set to `Visual Studio`.
2. Create a new `.cpp` file.
3. Enter the following content:

```c++
int main()
{
// 我能吞下玻璃而不伤身体。
int i_can_eat_glass_and_it_does_not_hurt_me;
return 0;
}
```

You can replace the Chinese comment by any text that contains non-ASCII characters.

4. Save the file. It will become:

```c++
int main()
{
    // 我能吞下玻璃而不伤身体。
int i_can_eat_glass_and_    it_does_not_hurt_me;
return 0;
}    
```

5. When `clang_format_style` is set to `Emulated Visual Studio`, the formatting is correct:

```c++
int main()
{
    // 我能吞下玻璃而不伤身体。
    int i_can_eat_glass_and_it_does_not_hurt_me;
    return 0;
}
```

**Expected behavior**

See the correct formatting above.

<details>
  <summary><strong>Logs</strong></summary>

```
Insert logs here.
```
</details>

**Screenshots**

**Additional context**

**Comment by sean-mcmanus** [comment]
Might be the same issue as https://github.com/microsoft/vscode-cpptools/issues/5914 .

**Comment by michelleangela** [comment]
I can only reproduce this when the comment above the variable includes non-ASCII characters. This may be the same issue as https://github.com/microsoft/vscode-cpptools/issues/5939

**Comment by michelleangela** [comment]
This is duplicate of https://github.com/microsoft/vscode-cpptools/issues/5914

**Comment by sean-mcmanus** [comment]
This is fixed with 0.30.0-insiders2: https://github.com/microsoft/vscode-cpptools/releases/tag/0.30.0-insiders2 .
